frontends/web/src/components/copy/Copy.tsx

```diff
@@ -27,10 +27,11 @@ type TProps = {
   className?: string;
   disabled?: boolean;
   flexibleHeight?: boolean;
+  name?: string;
   value: string;
 };
 
-export const CopyableInput = ({ alignLeft, alignRight, borderLess, value, className, disabled, flexibleHeight }: TProps) => {
+export const CopyableInput = ({ alignLeft, alignRight, borderLess, value, className, disabled, flexibleHeight, name }: TProps) => {
```

Comment:
nit: I started to use newlines if a function has more than 2-3 arguments, or if they have defaults assigned, to improve readability. But feel free to keep as is. Maybe one day I'll try to find a linter for this.